.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -90,37 +91,55 @@ jobs:
         with:
           branch_version_tag: ${{ needs.generate-tags.outputs.dev_tag }}
 
+      - name: Publish Dev Jars
```

Comment:
might be missing something, why move the jar publishing to be before the Docker image publishing?

Reply:
it doesn't actually matter, but I just wanted to try splitting up the steps into different jobs.  To do this, the publish step depends on some java setup steps in the Build Branch step, so I had to keep these within the same job, whereas the docker stuff I could split out into a separate job.

Reply:
understand, thanks.